### ISSUE TITLE
Why are you using dropout for validation steps?

### ISSUE THREAD

**Issue by kosonocky** [IBM/molformer#8]
https://github.com/IBM/molformer/blob/1ae8d842695d9b412fb937385ef93f05a359162e/finetune/finetune_pubchem_light.py#L251C1-L252C1

AFAIK, dropout shouldn't be use to get validation stats on your model, and you're using 0.1 dropout for this. Could you elaborate? I may be missing something.

**Comment by subhalingamd** [comment]
Hi. I thought I could write a brief answer in case you are still looking for it.

Dropouts are not used in `.eval()` mode and pytorch-lightning runs validation steps in eval() mode under the hood. so its equivalent to not having them. 

Hope it helps! Cheers.

**Comment by kosonocky** [comment]
@subhalingamd Wow I had no idea, thanks for clarifying. Makes total sense.

Cheers
